gitlab/clients/httpx_client.py

```diff
@@ -0,0 +1,11 @@
+from typing import Optional
+
+import httpx
+
+
+class Httpx_Client:
```

Comment:
I would follow camelcase and make them private here, so people know it's an internal implementation only :)

```suggestion
class _HttpxClient:
```

Reply:
Done